mqlc/builtin.go

```diff
@@ -344,3 +410,39 @@ func availableFields(c *compiler, typ types.Type) map[string]llx.Documentation {
 
 	return res
 }
+
+// We could have just as well used the `resources.Schema` here.
+// However, semantically t
```

Comment:
there is something missing with the comment

Reply:
ouch, who did that :joy: :see_no_evil: